kotsadm/operator/pkg/client/client.go

```diff
@@ -231,38 +231,41 @@ func (c *Client) registerHandlers(socketClient *socket.Client) error {
 
 		log.Println("received a deploy request for", args.AppSlug)
 
-		var applyResult, helmResult *commandResult
-		var applyError, helmError error
+		var deployRes *deployResult
+		var dryRunResult, applyResult, helmResult *commandResult
```

Comment:
can we move `dryRunResult` and `applyResult` to be inside the deferred function so that it's a bit clearer?